### PR TITLE
NOISSUE - Upgrade Go version to 1.13 in container images

### PR DESCRIPTION
**Pull request by blokovi**
Signed-off-by: Ivan Milošević <iva@blokovi.com>

### What does this do?
Upgrade Go to ver. 1.13 in container images

### Which issue(s) does this PR fix/relate to?
No issue

### CODE REVIEW

**Review by anovakovic01 — DISMISSED** [review]
LGTM!

**Review by drasko — CHANGES_REQUESTED** [review]
This cannot be accepted before CI is fixed as well. I.e. this PR must contain changes in CI script.

File: scripts/ci.sh [hunk 1]
```diff
@@ -1,6 +1,25 @@
 # This script contains commands to be executed by the CI tool.
 NPROC=$(nproc)
 
+update_go() {
+	echo "Update go version..."
```
Comment (dborovcanin) [inline scripts/ci.sh]: You can add check if the current version is 1.13 or later so that we can skip this step, which will speed up the CI.

Comment (blokovi) [reply scripts/ci.sh]: Added version comparison before upgrade on CI

**Review by dborovcanin — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**